healthcheck/src/healthcheck/status/auth.py

```diff
@@ -22,6 +22,7 @@ async def authenticate() -> Result[Token]:
         f"{ZIMFARM_API_URL}/auth/authorize",
         method="POST",
         payload={"username": ZIMFARM_USERNAME, "password": ZIMFARM_PASSWORD},
+        check_name="zimfarm-api-authentication",
     )
     return Result(
```

Comment:
For consistency, we miss a WARNING level log here when response.success is False

Reply:
This particular check hooks into `query_api` which logs warning already if there's no success

Reply:
Indeed